docs/src/main/paradox/reference.md

```diff
@@ -406,6 +428,15 @@ Returns a `tile` column containing the element-wise inequality of `tile1` and `r
 
 Returns a `tile` column with cell values of 1 where the `tile` cell value is in the provided array or list. The `array` is a Spark SQL [Array][Array]. A python `list` of numeric values can also be passed.
 
+### rf_local_extract_bits
+
+    Tile rf_local_extract_bits(Tile tile, Int start_bit, Int num_bits)
+    Tile rf_local_extract_bits(Tile tile, Int start_bit)
+
+Extract value from specified bits of the cells' underlying binary data. Working from the right, the bits from `start_bit` to `start_bit + num_bits` are extracted from cell values of the `tile`. The `start_bit` is zero indexed. If `num_bits` is not provided, a single bit is extracted.
+
+A common use case for this function is covered by @ref:[`rf_mask_by_bits`](reference.md#rf-mask-by-bits).
```

Comment:
I'm really pleased that you broke this whole process down to a set of more fundamental operations. Exactly the approach we want to take, and it's not always obvious to me what those are.